### ISSUE TITLE
A window ”This feature is not available without MATLAB running.“ keep pop up

### ISSUE THREAD

**Issue by vhwzx** [mathworks/MATLAB-extension-for-vscode#28]
I didn't install matlab on my computer and I just need to edit some .m file without running them, but when I type ctrl,  a window  "This feature is not available without MATLAB running." always pop up. 
Is there any way to disable those feature or stop window from poping up?
Thanks!

**Comment by WillFeavyour** [comment]
This also appears when MATLAB is installed but the "Matlab Connection Timing" setting is set to "never", i.e. MATLAB is not running in the background. 

Please add more details to the error popup (what is "_this_ feature"?) and the option to disable the popup when MATLAB is intentionally not running.

**Comment by Lachcim** [comment]
As a workaround, navigate to ~/.vscode-server/extensions/mathworks.language-matlab-1.1.4/out/extension.js and comment out these lines:

```javascript
client.onNotification(Notification.MatlabLaunchFailed, () => handleMatlabLaunchFailed());
client.onNotification(Notification.MatlabFeatureUnavailable, () => handleFeatureUnavailable());
client.onNotification(Notification.MatlabFeatureUnavailableNoMatlab, () => handleFeatureUnavailableWithNoMatlab());
```
Reload the window and the annoying notifications won't appear anymore.

Am I really the only one who installed this extension just for the syntax highlighting? I don't want to actually run MATLAB.

**Comment by hamidingit** [comment]
> As a workaround, navigate to ~/.vscode-server/extensions/mathworks.language-matlab-1.1.4/out/extension.js and comment out these lines:
> 
> ```js
> client.onNotification(Notification.MatlabLaunchFailed, () => handleMatlabLaunchFailed());
> client.onNotification(Notification.MatlabFeatureUnavailable, () => handleFeatureUnavailable());
> client.onNotification(Notification.MatlabFeatureUnavailableNoMatlab, () => handleFeatureUnavailableWithNoMatlab());
> ```
> 
> Reload the window and the annoying notifications won't appear anymore.
> 
> Am I really the only one who installed this extension just for the syntax highlighting? I don't want to actually run MATLAB.

@Lachcim , have you considered changing the setting under **MATLAB: Matlab Connection Timing** to "**never**"? That setting will make extension to not start MATLAB on your system and hence you should not see those errors.

**Comment by Lachcim** [comment]
@hamidingit Did you read the previous two comments? This is exactly what we tried and this is exactly what isn't working.

I went the extra mile for you and tracked down the source of this bug. Turns out that the notification is sent by the `handleDefOrRefRequest` function in [mathworks/MATLAB-language-server](https://github.com/mathworks/MATLAB-language-server). This in turn is called by `Connection.onDefinition`  when the user presses CTRL while the text cursor is hovering over a symbol. The notification keeps popping up whenever the user presses CTRL+S to save, hence the annoyance.

You guys will have to take over from here.
